### PR TITLE
custom seed usage, moved around production and development seed files, a...

### PR DESCRIPTION
**Pull request by bwasmith**
instructions for reference. default video has 5 videos

http://stackoverflow.com/questions/19872271/adding-a-custom-seed-file

### CODE REVIEW

File: db/seeds/default_video.rb [hunk 1]
```diff
@@ -0,0 +1,8 @@
+video1 = Video.create(:description => 'math, learn it', :title => 'Calculus AB', :youtube_id => 'bHNczNvOnGc')
+video2 = Video.create(:description => 'this is an asian film', :title => 'Asia and a Motorbike', :youtube_id => 'yJRy9o8gv6c')
+video3 = Video.create(:description => 'darren and naseem and brendan in bechtel', :title => 'You know it', :youtube_id => '6joOVjEemh4')
+video4 = Video.create(:description => 'potato', :title => 'Again and a Again', :youtube_id => '6joOVjEemh4')
+video5 = Video.create(:description => 'blah', :title => 'Again', :youtube_id => '6joOVjEemh4')
+
+
```
Comment (DarryQueen) [inline db/seeds/default_video.rb]: Unnecessary extra line endings. Keep one so Git doesn't complain; the reset should go.

File: db/seeds/default_video.rb [hunk 2]
```diff
@@ -0,0 +1,8 @@
+video1 = Video.create(:description => 'math, learn it', :title => 'Calculus AB', :youtube_id => 'bHNczNvOnGc')
+video2 = Video.create(:description => 'this is an asian film', :title => 'Asia and a Motorbike', :youtube_id => 'yJRy9o8gv6c')
+video3 = Video.create(:description => 'darren and naseem and brendan in bechtel', :title => 'You know it', :youtube_id => '6joOVjEemh4')
```
Comment (DarryQueen) [inline db/seeds/default_video.rb]: Cute.

File: lib/tasks/custom_seed.rake [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+namespace :db do
+  namespace :seed do
+    Dir[File.join(Rails.root, 'db', 'seeds', '*.rb')].each do |filename|
+      task_name = File.basename(filename, '.rb').intern #intern is alias for .to_sym  
+      task task_name => :environment do
+        load(filename) if File.exist?(filename)
+      end
+    end
+  end
+end
```
Comment (DarryQueen) [inline lib/tasks/custom_seed.rake]: Add a newline so this ugly red arrow can leave us alone.

**Comment by DarryQueen** [comment]
See comments. Will test in the morning.

**Comment by nalnaji** [comment]
These should now all be updated to include subject tags.

**Comment by DarryQueen** [comment]
Also consider squashing commits that should be grouped together. Effectively, each commit should be an abstracted feature.

**Comment by DarryQueen** [comment]
Cool. I suggest you squash into two commits: one for the file namespace seed thing, another for the seeds themselves.

**Comment by DarryQueen** [comment]
LGTM, assuming that this works.

**Pull request merged**